Add release notes for 20th of June and 7th of August (#7)
geniusyield/release-notes: releases.md

# Release 7th of August 2024

> - Trading Bot Analytics V1: A brand new part of the Genius Yield Platform where users can view the performance of their own and others trading bots.

Reviewer: ```suggestion
- Trading Bot Analytics V1: A brand new feature of the Genius Yield Platform where users are able to register and view the performance of their own and other trading bots.
```

- Added $OMNI, $DEDI, $SKY and $MOGI to the list of available tokens to trade.
- Removed warning label on Lace wallet as it is no longer experiencing issues.

# Release 20th of June 2024

- Added $MNT, $C4, $CLARITY, $CERRA, $rsRSN, $rsBTC, $ANGELS and $NIKE to the list of available tokens to trade.
- Updated $USDM icon.

# Release 3rd of June 2024

- Suggestion to place a better market order when detected by our systems.
- Automatically select ADA as opposite asset when changing 'from' token.

# Release 29th of May 2024

- Added $SPLASH token to the list of available tokens to trade.

> - Added warning to users that the Lace wallet is experiencing issues.

Author: Changed some naming to past tense
